cli/azd/test/functional/cli_test.go

```diff
@@ -66,8 +66,14 @@ func Test_CLI_Version_PrintsVersion(t *testing.T) {
 
 	rn := os.Getenv("GITHUB_RUN_NUMBER")
 	if rn != "" {
-		// TODO: This should be read from cli/version.txt (https://github.com/Azure/azure-dev/issues/36)
-		require.Contains(t, out, "0.2.0")
+		_, filename, _, gotFilePath := runtime.Caller(0)
```

Comment:
@danieljurek  does it makes sense to use `CLI_VERSION` since the CI is already orchestrating that? That way we avoid the number of things that knows about `cli/version.txt`